libraries/drivers_webserver_base.rb

```diff
@@ -80,12 +80,16 @@ def add_appserver_config
         opts = { application: app, deploy_dir: deploy_dir(app), out: out, conf_dir: conf_dir, adapter: adapter,
                  name: Drivers::Appserver::Factory.build(context, app).adapter }
         return unless Drivers::Appserver::Base.adapters.include?(opts[:name])
+        generate_appserver_config(opts, site_config_template, site_config_template_cookbook)
+      end
 
+      def generate_appserver_config(opts, source_template, source_cookbook)
```

Comment:
Why passing separate variables here? Can we stick to using `opts` only?

Reply:
`opts` get passed to the cookbook template itself at L93, and I figured that the cookbook template didn't need to know the `source_template` and `source_cookbook` values to do its work.